### ISSUE TITLE
Internationalize the app

### ISSUE THREAD

**Issue by mikehearn** [vinumeris/lighthouse#14]
Currently there's no support for translating the UI into other languages, handling different ways of writing numeric amounts etc. Probably we should do this using the Java version of GNU gettext, as the native Java i18n api is kind of old and not as good as gettext IMO.

**Comment by dabura667** [comment]
+ 1 willing to translate into Japanese.

**Comment by dnivi3** [comment]
Absolutely! I'd be willing to translate it into Norwegian!

**Comment by dabura667** [comment]
transifex or crowdin are great platforms to manage the translations.

They offer free accounts / hosting for open source projects.
(I know @mikehearn knows this, but just for anyone else who might view this recommendation)

**Comment by mikehearn** [comment]
Yep, thanks. Before it's possible to do translations of the app though, the app itself needs to be "internationalized". It currently isn't. That would take a fair amount of work.

**Comment by mikehearn** [comment]
Thanks to @digital-dreamer the app is now (mostly) internationalized! The project is on Transifex under the name "lighthouse-app" and we begin with a Czech translation.

If you would like to contribute, please go sign up for transifex.com and check it out:

https://www.transifex.com/projects/p/lighthouse-app/resource/lighthousepot/

There are 216 strings to translate but most of them should be short/easy. If you find strings that are confusing or hard to translate, let us know.
